app/assets/stylesheets/common/components/more-topics.scss

```diff
@@ -39,14 +39,18 @@
     .topic-list-header:after {
       content: "";
       display: block;
-      height: 10px;
+      height: 12px;
```

Comment:
Added 2 pixels, so padding-top: 0 on td could be cancelled out exactly (appears no changes to topic list when not selected)